app/concerns/long_runnable.rb

```diff
@@ -117,6 +121,7 @@ def boolify(value)
     end
 
     private
+
```

Comment:
The good old battle over the empty space after the `private` modifier :smile: Maybe we should link the style guide (github or bbatsov) which huginn follows in the contributing section?